README.rst

```diff
@@ -31,58 +31,64 @@ Installation
 
 .. code-block:: console
 
-    (<custom virtualenv>)$ pip install invenio-cli
+    $ pip install invenio-cli
 
 Usage
 =====
 
-
 Local Development environment
 -----------------------------
 
 .. code-block:: console
 
     # Initialize environment and cd into <created folder>
-    (<custom virtualenv>)$ invenio-cli init --flavour=RDM
-    (<custom virtualenv>)$ cd <created folder>
+    $ invenio-cli init --flavour=RDM
+    $ cd <created folder>
 
-    # Install python dependencies, link/copy assets + statics, install js
+    # Install locally
+    # install python dependencies, link/copy assets + statics, install js
```

Comment:
Doubt: assets + statics first they are copied/linked and then built? It's not all done in one command right? is like create + build?

Reply:
Right, it does create + install + build behind the scenes.
 
1- the invenio modules' statics/assets are collected first which creates/fills the `assets/` and `static/` folders in the instance path (collect and webpack create). 
2- Then the js dependencies are installed optionally, because we don't always want to call `npm install` which is slow, especially if we just changed some local file. See #72 
3- Then the *local assets* are copied / symlinked in the instance path, so that *all* the assets are in the same location
4- finally webpack build is run, which will build all the assets because they are now all present in the same `<instance_path>/assets/` folder.